include/nd_memory.h

```diff
@@ -89,6 +115,39 @@ inline void for_each_contiguous_chunk(const nd_copy_layout& layout, F&& f) {
 	}
 }
 
+/// From allocation `source_base` spanning `source_box` to allocation `dest_base` spanning `dest_box`, copy `copy_region` elements of `elem_size` bytes.
```

Comment:
Should be `source_layout` and `dest_layout`?

Reply:
fixed.